src/graphics/program-lib/chunks/ambientPrefilteredCube.frag

```diff
@@ -4,7 +4,11 @@ uniform samplerCube texture_prefilteredCubeMap4;
 #endif
 
 void addAmbient() {
+#ifdef CUBEMAPROT
+    vec3 fixedReflDir = fixSeamsStatic(dNormalW*cubeMapRotationMatrix, 1.0 - 1.0 / 4.0);
+#else
     vec3 fixedReflDir = fixSeamsStatic(dNormalW, 1.0 - 1.0 / 4.0);
+#endif
     fixedReflDir.x *= -1.0;
```

Comment:
can we remove this line which flips the .x of reflection vector? TO use dynamic cubemaps, this line needs to not be applied, and I was hoping to handle it by using the matrix. Ideally we add this flip into the matrix on the cpu and expose it as a material option, so that it can be disabled for realtime cubemaps.

Reply:
Since we don't want to add an extra matrix multiply to all shaders, it probably makes sense for `skyboxRotation` to be exposed as 3x3 matrix instead of quaternion so you can scale as necessary.

Reply:
Switched from Quat to Mat3 and embedded the -x flip in the Matrix - also moved the cube map rotation property to pc.Texture. I had to change a lot of shader chunks in order to embed the -x flip and keep identical behavior for reflection and ambient shader chunks that do not use cube maps. Its quite likely to break some custom shader chunks...